src/App.tsx

```diff
@@ -1,17 +1,74 @@
+import React, { useEffect, useState } from 'react';
+
 import { AboutComponent } from './components/about/About.component.tsx';
 import { FooterComponent } from './components/footer/Footer.component.tsx';
 import { HeaderComponent } from './components/header/Header.component.tsx';
+import { ProducstList } from './components/productsList/ProductsList.component.tsx';
 
 import styles from './App.module.css';
 
+type Page = string | null;
```

Comment:
Better to move it into another folder with interfaces

Reply:
Done.